counter-sink/src/main/java/org/springframework/cloud/stream/module/metrics/CounterSinkConfiguration.java

```diff
@@ -41,17 +36,6 @@
 
     private EvaluationContext evaluationContext;
 
-    @Bean
-    public CounterService counterService(RedisMetricRepository redisMetricRepository) {
```

Comment:
Yes, although there is still some magic that eludes me (seems like MetricRepository gets a different treatment from MetricReader/Writer for example)